taxi/urls.py

```diff
@@ -42,10 +43,19 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path("cars/<int:pk>/assign/", assign_driver, name="assign-driver"),
+    path("cars/<int:pk>/remove/", remove_driver, name="remove-driver"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path("drivers/<int:pk>/delete/",
+         DriverDeleteView.as_view(),
+         name="driver-delete"),
```

Comment:
Code Style: Use '-' to split words in URL identification parameter `name`, not the '_'. So it should be 'driver-delete' instead of 'driver_delete'.